scipy/spatial/distance.py

```diff
@@ -1084,7 +1084,7 @@ def pdist(X, metric='euclidean', p=2, w=None, V=None, VI=None):
 
        .. math::
 
-            d(u,v) = \\frac{\\sum_i {u_i-v_i}}
+            d(u,v) = \\frac{|\\sum_i {u_i-v_i}|}
```

Comment:
and the denominator also needs the same ``| |`` added to it. I see that most references don't have that, but our implementation does. Shouldn't matter because this metric should only be used with positive values, but still better to match docs to implementation.